doc/swss-schema.md

```diff
@@ -640,11 +640,11 @@ Equivalent RedisDB entry:
                                                 ; If "system" warm start knob is true, docker level knob will be ignored.
                                                 ; If "system" warm start knob is false, docker level knob takes effect.
 
-    timer_name      = 1*255VCHAR,               ; timer_name is the name of warm start timer for the whole system or the specific docker,
-                                                ; Ex. "neighbor_timer", "bgp_timer". The name should not contain "," character.
-                                                ; There could be more than one timer in a docker or the system, separated by ",".
-
-    timer_duration  = 1*4DIGIT,                 ; timer duration, in seconds. Separated by "," if there are more than on timer.
+    app_timer       = 1*10DIGIT,                ; app_timer is the abstract name of warm start
+                                                ; timer for the whole system or the specific docker.
+                                                ; Real name is "neighsyncd_timer", "bgp_timer" etc.
+                                                ; There could be more than one timer in a docker or the system.
```

Comment:
This seems contradictory to the existing practice in swss-schema.md. For all other cases, the field name is fixed and value section is dynamic.  

We need to make sure the pros outweigh the cons and it complies with ABNF spec.

Reply:
This is not meant for dynamic field name, it will be a fixed one if we know the application name which requires warm-start timer.  Maybe I should just put "neighsyncd_timer" and "bgp_timer" here, and we can add more later if needed, I was trying to describe what the name typically looks like.

Reply:
Can you please point me to a design document?